pkg/kubectl/cmd/util/helpers.go

```diff
@@ -255,6 +255,28 @@ func MultilineError(prefix string, err error) string {
 	return fmt.Sprintf("%s%s\n", prefix, err)
 }
 
+// PrintErrorWithCauses prints an error's kind, name, and each of the error's causes in a new line.
+// The returned string will end with a newline.
+// Returns true if a case exists to handle the error type, or false otherwise.
+func PrintErrorWithCauses(err error, errOut io.Writer) bool {
+	switch t := err.(type) {
+	case *kerrors.StatusError:
+		errorDetails := t.Status().Details
+		fmt.Fprintf(errOut, "error: %s %q is invalid\n", errorDetails.Kind, errorDetails.Name)
```

Comment:
Thanks for catching this, since the message `error: ... is invalid` is also part of the output in the loop right below it, I placed it in the already existing check for a `nil` errorDetails